mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -823,6 +825,12 @@ class WidgetAdapter(
             } else {
                 group.check(checkedId)
             }
+
+            // Layout has to be drawn before measuring the width
+            handler.post {
+                group.isVisible = group.measuredWidth < (parent.width * 0.8)
+                spinner.isVisible = !group.isVisible
```

Comment:
No further spinner setup needed here? Hardly can believe that...

Reply:
The spinner setup is done in the super class: `SelectionViewHolder`